managed/ui/src/components/universes/UniverseForm/ClusterFields.js

```diff
@@ -1304,7 +1321,7 @@ export default class ClusterFields extends Component {
         <Field
           name={`${clusterType}.enableClientToNodeEncrypt`}
           component={YBToggle}
-          isReadOnly={isFieldReadOnly}
+          isReadOnly={ this.clientToNodeEncryptField(isFieldReadOnly, this.state.enableNodeToNodeEncrypt)}
```

Comment:
Is the intention to only set the field initially? What about when `this.state.enableNodeToNodeEncrypt` changes value? Shouldn't the client-to-node field then become editable?

Reply:
I've updated the Doc against the same.